server/admin_management/requirements.txt

```diff
@@ -1,10 +1,10 @@
-fastapi==0.104.1
-uvicorn[standard]==0.24.0
-SQLAlchemy==2.0.23
+fastapi==0.108.0
+uvicorn[standard]==0.25.0
+SQLAlchemy==2.0.25
 mangum==0.17.0
 boto3==1.28.83
-psycopg2-binary==2.9.7
+psycopg2-binary==2.9.9
 python-jose==3.3.0
-httpx==0.24.1
-pydantic==2.4.2
-pydantic_core==2.10.1
\ No newline at end of file
+httpx==0.26.0
+pydantic==2.5.3
+pydantic_core==2.14.6
```

Comment:
Normally I would be very cautious for `SQLAlchemy` and `pydantic` and `pydantic_core` upgrade and separate them into different PRs. They could have very hidden behavior change that's hard to test or spot right away. So in case anything that were not caught initially from PR but discovered later in DEV or TEST environment we could potentially identify it or fix it easily.

Reply:
I agree, I don't like to have these many updates in one pr, even they are all minor ones. Maybe later we can see if  renovate can separate them based on directories.